README.md

```diff
@@ -317,3 +326,48 @@ SYCL and the SYCL logo are trademarks of the Khronos Group Inc.
 [lesson-10-source]: ./Code_Exercises/Exercise_10_Managing_Dependencies/source.cpp
 [lesson-10-solution]: ./Code_Exercises/Exercise_10_Managing_Dependencies/solution.cpp
 
+[lesson-11-slides]: ./Lesson_Materials/Lecture_11_In_Order_Queue/
+[lesson-11-exercise]: ./Code_Exercises/Exercise_11_In_Order_Queue/doc.md
+[lesson-11-source]: ./Code_Exercises/Exercise_11_In_Order_Queue/source.cpp
+[lesson-11-solution]: ./Code_Exercises/Exercise_11_In_Order_Queue/solution.cpp
+
+[lesson-12-slides]: ./Lesson_Materials/Lecture_12_Temporary_Data/
+[lesson-12-exercise]: ./Code_Exercises/Exercise_12_Temporary_Data/doc.md
+[lesson-12-source]: ./Code_Exercises/Exercise_12_Temporary_Data/source.cpp
+[lesson-12-solution]: ./Code_Exercises/Exercise_12_Temporary_Data/solution.cpp
+
+[lesson-13-slides]: ./Lesson_Materials/Lecture_13_Load_Balancing/
```

Comment:
I think we should keep this one as "Lecture_13_Multiple_Devices" as it talks about using multiple devices but load balancing is just one usage of this.

Reply:
Change made